### PR TITLE
fix: don't select suggestions on hover

### PR DESCRIPTION
**Pull request by olchyk98**
Fixes the unnecessary behaviour and scrolling issue.

### CODE REVIEW

File: lib/choicesDropdown.js [hunk 1]
```diff
@@ -530,18 +530,18 @@ export default class ChoicesDropdown {
...
 
-    optionNode.addEventListener('mouseenter', ({ target }) => {
-      this.hoverOption(target)
-    })
-
     optionNode.addEventListener('click', () => {
       this.addValue(value, key, true)
     })
 
+    // optionNode.addEventListener('mouseenter', ({ target }) => {
```
Comment (orestisrodriguez) [inline lib/choicesDropdown.js]: ?

Comment (olchyk98) [reply lib/choicesDropdown.js]: I've done some research and most of the dropdown libraries **don't** allow user to select suggestions using mouse. Suggestions can be selected only using the keyboard. Even the mt-select component follows the same behavior. Therefore, this feature is unnecessary and, on my opinion, should be deprecated.

Comment (olchyk98) [reply lib/choicesDropdown.js]: It's kind of confusing. By "select" I mean hover aka mouseenter.

Comment (orestisrodriguez) [reply lib/choicesDropdown.js]: sure, remove it fully from the code then :)

**Comment by orestisrodriguez** [comment]
`Uncaught TypeError: Cannot read property 'classList' of null` when selecting a lot of suggestions fast

**Pull request merged**